packages/dd-trace/src/appsec/reporter.js

```diff
@@ -204,7 +207,6 @@ function finishRequest (req, res) {
     rootSpan.addTags(Object.fromEntries(metricsQueue))
 
     keepTrace(rootSpan, ASM)
-
     metricsQueue.clear()
```

Comment:
accidental diff i guess
```suggestion
    keepTrace(rootSpan, ASM)

    metricsQueue.clear()
```